Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -244,6 +274,10 @@
     Telecrystal: 65
   categories:
     -  UplinkWeaponry
+  conditions:
+  - !type:BuyerSpeciesCondition
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)